repos/system_upgrade/el7toel8/actors/removeupgradebootentry/tests/unit_test.py

```diff
@@ -1,7 +1,7 @@
 import pytest
 
 from leapp.exceptions import StopActorExecutionError
-from leapp.libraries import stdlib
+from leapp.libraries import stdlib  # noqa: F401
```

Comment:
```suggestion
from leapp.libraries.stdlib import api
```

and later in the tests there should be `setattr(api, 'consume', ....` and `setattr(api, 'produce', ...`